go.mod

```diff
@@ -170,36 +169,36 @@ replace (
 	github.com/openshift/library-go => github.com/mhenriks/library-go v0.0.0-20230310153733-63d38b55bd5a
```

Comment:
`openshift/api` (looks like `release-4.15` is the appropriate branch for k8s 1.28) and `openshift/library-go` should probably be updated as well.  library-go is tricky as this was never merged:  https://github.com/openshift/library-go/pull/1486

This is obviously pretty annoying.  I guess I'll update my fork again for now

Reply:
Let me know when that is done and I can update both

Reply:
Looks like last week the change was made on master and backported to release-4.16 in another PR (https://github.com/openshift/library-go/commit/434fa21f5e457d51f436c4b3804c7803db1da42d)

So if you want to go straight to 1.29 you can use release-4.16 branch in library-go
If you insist on doing 1.28 first (sounds like more work to me) then you can use `github.com/mhenriks/library-go export-certrotation-4.15`

Reply:
We can't - https://github.com/openshift/library-go/blob/master/go.mod#L3, same for ctrl runtime

Reply:
Do you mean a few separate PRs to bump go/update builder image, right?

Reply:
Anyway, this is huge as it is, if more (1.28->1.29) adaptations are needed this PR will just be all over the place
so I prefer to take the 1.28->1.29 stuff out